crates/loro-wasm/tests/tree.test.ts

```diff
@@ -222,6 +222,28 @@ describe("loro tree node", () => {
     unsub()
     assertEquals(child2.parent()!.id, child.id);
   });
+
+  it("toJSON", () => {
```

Comment:
It feels like a good place to use snapshot tests. https://vitest.dev/guide/snapshot

It'll be better to also have some tests for attached LoroTree